Wootric/WootricSDK-Android
# CES/CSAT support

This PR adds CES/CSAT to the Android SDK to match how things
are working with web and the iOS SDK.

## Changes

- Adds CES/CSAT constants
- Changes LocalizedTexts to obtain ces/csat anchors
- Adds surveyType to settings
- Updates UI of smartphone and tablets

## Trello

https://trello.com/c/ZeXa9aM3/1450-2-5-csat-ces-android-sdk

## How to test

- Pull this branch
- Set client id, client secret and account token for ces, csat and nps account
- UI should show correspondent scale
- Test that it correctly identifies promoters, passives and detractors for each mode
